tools/python/boutiques/tests/test_export.py

```diff
@@ -90,7 +90,7 @@ def test_export_json_obj(self):
         ref_name_p2 = "example1_docker_exported_doi_python2.json"
         ref_file_p2 = opj(example1_dir, ref_name_p2)
         self.assertFalse(bosh(["export", "carmin", example1_desc_doi, fout]))
-        result = open(fout, "r").read().strip()
-        assert(result == open(ref_file, "r").read().strip() or
-               result == open(ref_file_p2, "r").read().strip())
+        result = open(fout, "U").read().strip()
+        assert(result == open(ref_file, "U").read().strip() or
+               result == open(ref_file_p2, "U").read().strip())
         os.remove(fout)
```

Comment:
This all looks great to me. There are probably a few more non-portable pieces of code in the project now, fixing them isn't super high priority, but it's definitely a good idea to fix them whenever we have a chance!

Reply:
This only made my radar as my tests stopped passing when I incorporated the new JSON from the develop branch. My thought process was that the linebreak wars are dumb and that eventually systems should silently consume the difference painlessly, vs. adding process to developers (like running dos2unix on code before commit, which I am considering). 

The Universal newline was an approach in that direction, I suspect, but it became deprecated because in later python it just became the default behaviour. Semantically, the Linux is approach is correct and succinct. Syntactically the Windows approach is literal, historical and also reasonably correct. Mac was wrong. Someday, when everyone has forgotten the war, it will end.

Reply:
Notes from the meeting today:
1. Look into decoding directly to UTF-8.
2. Make sure the characters are properly substituted on display.

Reply:
How about doing this upon parsing?
```
if isinstance(v, unicode):  # this is only true in Python 2
                v = v.encode("utf-8")  # now v is a string
```

Reply:
This does not solve the underlying problem which I've narrowed to terminal support of utf-8. I think it might also be an error in python 3 as unicode is no longer defined `NameError: name 'unicode' is not defined`.

I've filled out my description in the PR comment. The question will be whether we want to check for UTF-8 support and only conditionally do the uglier encoding, or if we want it consistent (and ugly).